ote/src/cljs/taxiui/views/front_page.cljs

```diff
@@ -0,0 +1,60 @@
+(ns taxiui.views.front-page
+  "Front page for Taxi UI"
+  (:require [clojure.string :as str]
+            [ote.localization :refer [tr tr-key]]
+            [ote.theme.colors :as colors]
+            [ote.ui.common :refer [linkify]]
+            [ote.app.utils :refer [user-logged-in?]]
+            [re-svg-icons.feather-icons :as feather-icons]
+            [stylefy.core :as stylefy]
+            [taxiui.styles.front-page :as style-front-page]))
+
+(let [host (.-host (.-location js/document))]
+  (def test-env? (or (str/includes? host "test")
+                     (str/includes? host "localhost"))))
+
+(defn test-env-warning []
+  [:div.test-env-warning
+   {:style {:margin "0.2em"
+            :border "4px dashed red"}}
+   [:p {:style {:margin "10px 0px 0px 10px"
+                :font-weight "bold"}}
+    "TÄMÄ ON TESTIPALVELU!!"]
```

Comment:
I assume this gets deleted before going to production :)

Reply:
This is the alert box shown in test environment, it is like that on purpose. Will be deleted on Taxi UI side though.